index.js

```diff
@@ -61,12 +70,7 @@ function prepareAndSendToken (req, res, type, token, options, cb) {
     params[type] = buffer.toString('base64');
 
     // construct the Signature: a string consisting of the concatenation of the SAMLResponse,
-    // RelayState (if present) and SigAlg query string parameters (each one URLencoded)
```

Comment:
Should we not at the contrary continue omitting RelayState from the signature ? With the code line 30, we seem to fix the fact that we send empty RelayState to Azure, so Azure doe snot include in the SIG, but here we are now including it on our side in the SIG ? 

Am I confused maybe ?